src/languages/javascript.js

```diff
@@ -384,7 +384,7 @@ export default function(hljs) {
     match: regex.concat(
       /\b/,
       noneOf([
-        ...ECMAScript.BUILT_IN_GLOBALS,
+        ...ECMAScript.BUILT_IN_GLOBALS.map((x) => x + '\\('),
```

Comment:
Don't we need this same protection for super and import also?  And wouldn't just adding a `\b` on the other side of `noneOf` get the job done here without explicitly looking for a `(`?

Reply:
yes, there might be more bugs in the way the parser handles keywords and other keywords affected. I haven't tested that and I'm not sure of what the logic behind the original code was, but your suggestion seems reasonable. 

Same thing for using `\b` instead of `(`, again I'm not sure of what the original logic was so I tried to find a minimal solution that would fix the test case I encountered.

Reply:
Happy to review again after you update.

Reply:
That PR was just a suggestion. I'll let someone with a better understanding of the codebase to contribute a fix.